searx/engines/duckduckgo.py

```diff
@@ -77,13 +77,12 @@ def cache_vqd(query, value):
         c.set(key, value, ex=600)
 
 
-def get_vqd(query):
+def get_vqd(query, reset_token):
```

Comment:
@jazzzooo thanks a lot for your perseverance and patience (I know myself how tiring this work can be) .. one nit I have: I think we do not need the `reset_token` argument, since all caller set it to `False` .. may I'm wrong, can you please have another look?

Reply:
@return42 yes no worries. The get_vqd function calls itself on line 109 with reset_token=True. This was my fix for non-redis instances. We get a captcha on a search unless we "reset" the token by sending a different query and then resending the original one. 

So to test: turn off redis, search something uncommon, get to the end of the results. Now search the same thing again. On master you get IndexError (because it's a guaranteed captcha). Now we send another query to get a new token, then send the same one again, it takes a second but no more crash and you get results.